packages/theme-patternfly-org/scripts/md/parseMD.js

```diff
@@ -277,9 +279,11 @@ module.exports = {
     globs.props.push({ glob, ignore });
     sync(glob, { ignore }).forEach(sourcePropsFile);
   },
-  sourceMD(glob, source, ignore) {
-    globs.md.push({ glob, source, ignore });
-    sync(glob, { ignore }).forEach(file => sourceMDFile(file, source));
+  sourceMD(buildMode) {
+    return function(glob, source, ignore) {
+      globs.md.push({ glob, source, ignore });
+      sync(glob, { ignore }).forEach(file => sourceMDFile(file, source, buildMode));
+    }
```

Comment:
Can you move this to `generate.js` instead? Something like:
```js
const sourceMDWithOptions = (...args) => sourceMD(...args, options._name);
```

Reply:
Sure. I went back and forth on doing this a few different ways, wasn't super happy with it anyway.